.flake8

```
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```

```suggestion
[flake8]
max-line-length = 160
```
I think just moving a part of long line to another line is good.
`ansible-test sanity` intentionally sets some limitations for readability